cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/polling/task/NoExceptionOnTimeoutClouderaManagerListenerTask.java

```diff
@@ -0,0 +1,22 @@
+package com.sequenceiq.cloudbreak.cm.polling.task;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.sequenceiq.cloudbreak.cluster.service.ClusterEventService;
+import com.sequenceiq.cloudbreak.cm.client.ClouderaManagerApiPojoFactory;
+import com.sequenceiq.cloudbreak.cm.polling.ClouderaManagerCommandPollerObject;
+
+public class NoExceptionOnTimeoutClouderaManagerListenerTask extends ClouderaManagerDefaultListenerTask {
```

Comment:
FYI: We are implementing something similar in CB-14422 for all pollers.

Reply:
A similar class already exists if I'm not mistaken, with a specific name. Hence introduced this.